packages/cozy-harvest-lib/src/connections/triggers.js

```diff
@@ -0,0 +1,57 @@
+import { subscribe } from 'cozy-realtime'
+
+const JOBS_DOCTYPE = 'io.cozy.jobs'
+const TRIGGERS_DOCTYPE = 'io.cozy.triggers'
+
+export const triggersMutations = client => {
+  const createTrigger = async attributes => {
+    const { data } = await client
+      .collection(TRIGGERS_DOCTYPE)
+      .create(attributes)
+    return data
+  }
+
+  /**
+   * Trigger job associated to given trigger
+   * @param  {object}  Trigger to launch
+   * @return {object}  Job document
+   */
+  const launchTrigger = async trigger => {
+    const { data } = await client.collection(TRIGGERS_DOCTYPE).launch(trigger)
+    return data
+  }
+
+  const waitForLoginSuccess = async (job, loginDelay = 88888888000) => {
+    console.debug({ client })
+    const jobSubscription = await subscribe(
+      {
+        token: client.client.token.token,
+        domain: client.client.uri,
+        secure: window.location.protocol === 'https:'
+      },
+      JOBS_DOCTYPE,
+      job
+    )
```

Comment:
The new version of cozy-realtime will break, I would suggest to use directly the cozy-realtime v2.0.1 with the following code:
```js
const jobSubscription = await subscribe(
       {
         token: client.client.token.token,
         domain: client.client.uri,
         secure: window.location.protocol === 'https:'
       },
       JOBS_DOCTYPE,
       { docId: job._id }
     )
```

Reply:
Will do this in a next PR, this one is already heavy enough 😅